R/stat-bin.r

```diff
@@ -1,5 +1,7 @@
 #' @param binwidth The width of the bins. Can be specified as a numeric value,
-#'   or a function that calculates width from x.
+#'   or a function that calculates width from unscaled x. When specifying a 
```

Comment:
The comma at the end of the preceding line (after "value") should be removed, and it would be slightly better to write "or *as* a function ..."

Reply:
Also, thinking about it some more, I'm not entirely sure what "unscaled x" means. Does it mean before any scale transformations have been applied? Maybe better to clarify.

Reply:
Thank you for taking the time to review. I solved the former issue in the most recent commit, but I want to make certain that the "unscaled x" portion is unambiguous before I go further. Would the text below be more clear? 

"or as a function that calculates width from x prior to any x-axis transformations."

Reply:
I think the text will read better if you leave "unscaled x" and then define what that means:
> ...unscaled x. Here, "unscaled x" refers to the original x values in the data, before application of any scale transformation.

Reply:
Wonderful! I will do exactly that.